TramsDataApi/Factories/EstablishmentResponseFactory.cs

```diff
@@ -94,7 +96,7 @@ public static EstablishmentResponse Create(
                 Ukprn = establishment.Ukprn,
                 FeheiIdentifier = establishment.Feheidentifier,
                 FurtherEducationType = establishment.FurtherEducationTypeName,
-                OfstedLastInspection = establishment.OfstedLastInsp,
+                OfstedLastInspection = string.Empty,
```

Comment:
Why empty?

Reply:
I was told by Sam to keep the fields but just return an empty string